jolly-jellyfish/src/static/filters.svg

```diff
@@ -0,0 +1,22 @@
+<!--
+    CSS Image filters for themes.
+-->
+
+<svg xmlns="http://www.w3.org/2000/svg" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink" width="0" height="0">
+    <filter id="red">
+        <feColorMatrix type="matrix" values="
+            1   0.6 0.6 0 1
+            0.3 0   0   0 0
+            0.3 0   0   0 0
+            0   0   0   1 0
+        "/>
```

Comment:
could you add a comment about what these values mean?

Reply:
Added